### PR TITLE
More timings of `linera` operations and description of the operations beings timed.

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

The timings that show are not precise enough and hard to parse.

## Proposal

The following is done:
* Each "Operation confirmed in X ms" is replaced by a more precise statement.
* More timings are added for other operations.

## Test Plan

The CI.

## Release Plan

This PR mostly impacts the log obtained by running the CI. This is useful for operators on the TestNet / DevNet.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

File: linera-service/src/linera/main.rs [hunk 1]
```diff
@@ -1609,40 +1715,57 @@ async fn run(options: &ClientOptions) -> Result<i32, anyhow::Error> {
...
 
             WalletCommand::SetDefault { chain_id } => {
+                let start_time = Instant::now();
                 options
                     .wallet()
                     .await?
                     .mutate(|w| w.set_default_chain(*chain_id))
                     .await??;
+                info!(
+                    "Setting default chain in {} ms",
```
Comment (ma2bd) [inline linera-service/src/linera/main.rs:1738]: That's not the best sentence in English (it should be in the past)

Comment (MathieuDutSik) [reply linera-service/src/linera/main.rs]: Ok, corrected. That is indeed important as it is visible to users.

File: linera-service/src/linera/main.rs [hunk 2]
```diff
@@ -968,7 +995,10 @@ impl Runnable for Job {
                     .await
                     .context("Failed to create application")?;
                 info!("{}", "Application published successfully!".green().bold());
-                info!("Time elapsed: {} ms", start_time.elapsed().as_millis());
+                info!(
+                    "Application published and created application in {} ms",
```
Comment (ndr-ds) [inline linera-service/src/linera/main.rs:999]:
```suggestion
                    "Application published and created in {} ms",
```

File: linera-service/src/linera/main.rs [hunk 3]
```diff
@@ -1569,38 +1644,63 @@ async fn run(options: &ClientOptions) -> Result<i32, anyhow::Error> {
...
                     full_storage_config.delete_all().await?;
+                    info!(
+                        "All namespaces deleted in {} ms",
+                        start_time.elapsed().as_millis()
+                    );
                 }
                 DatabaseToolCommand::DeleteNamespace { .. } => {
                     full_storage_config.delete_namespace().await?;
+                    info!(
+                        "Nnamespace deleted in {} ms",
```
Comment (ndr-ds) [inline linera-service/src/linera/main.rs:1659]:
```suggestion
                        "Namespace deleted in {} ms",
```

**Review by ndr-ds — APPROVED** [review]

File: linera-service/src/linera/main.rs (hunk 2 above)
Comment (MathieuDutSik) [reply linera-service/src/linera/main.rs:999]: Ok, done.

File: linera-service/src/linera/main.rs (hunk 3 above)
Comment (MathieuDutSik) [reply linera-service/src/linera/main.rs:1659]: Ok, done.
